Arpeggiator.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <iostream>
```

Comment:
is `<iostream`> used at all in this header file?
if it's not, don't include it.
Include it where it is used: in the cpp file.
- [ ] revise